scripts/subscribeEventProof.js

```diff
@@ -269,6 +272,28 @@ async function main (networkName, bridgeContractAddress) {
                 }
             })
         });
+
+    //listen to withdraw events on eth to update claimed proofs
+    pegContract.on("Withdraw", async (from, to, value, event) => {
```

Comment:
we had observed in past that WebSocket Infura provider does not listen to event (deposit) in claim relayer.
used Alchemy provider (https://github.com/cennznet/bridge-contracts/commit/052e2f136b556816bfdfcaee4bc51e2e2900a81f)

Reply:
oo okay sweet i'll switch over to that provider